### PR TITLE
Fix Styling

### PR DESCRIPTION
**Pull request by ycheng627**
Fix the styling of codebase by:
- Restoring app name
- Align comment styles
- Return on promise resolve/reject whenever possible
- Align function styling by using normal function for named function, and adding private/public modifiers
- Align all naming style by removing prefix underscore when possible

### CODE REVIEW

**Review by andy920262 — APPROVED** [review]

File: diagnostics-app/src/app/routine-v2/routine-v2-card/routine-v2-card.component.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export class RoutineV2CardComponent implements OnInit, OnDestroy {
   private _uuid?: string;
   // An integer between 0 ~ 100 detailing the percentage ran of the routine.
   private _percentage = 0;
-  // the state of the underlying routine
+  // The state of the underlying routine
```
Comment (BaiiYuan) [inline diagnostics-app/src/app/routine-v2/routine-v2-card/routine-v2-card.component.ts:61]: nit: missing period.

File: diagnostics-app/src/app/layout/content-layout/content-layout.component.it.spec.ts [hunk 2]
```diff
@@ -41,19 +41,19 @@ describe('integration: component ContentLayout', () => {
...
     drawerToggle.click();
     fixture.detectChanges();
     expect(sidenavComponent.opened).toBe(!defaultSidenavIsOpened);
 
-    // should flip the open state of the drawer
+    // Should flip the open state of the drawer
     drawerToggle.click();
     fixture.detectChanges();
-    // update handle
+    // Update handle
```
Comment (BaiiYuan) [inline diagnostics-app/src/app/layout/content-layout/content-layout.component.it.spec.ts:56]: nit: missing period.

**Review by BaiiYuan — APPROVED** [review]
LGTM with nits.

File: diagnostics-app/src/app/routine-v2/routine-v2-card/routine-v2-card.component.ts (hunk 1 above)
Comment (ycheng627) [reply diagnostics-app/src/app/routine-v2/routine-v2-card/routine-v2-card.component.ts:61]: Done.

File: diagnostics-app/src/app/layout/content-layout/content-layout.component.it.spec.ts (hunk 2 above)
Comment (ycheng627) [reply diagnostics-app/src/app/layout/content-layout/content-layout.component.it.spec.ts:56]: Done.

**Pull request merged**